### PR TITLE
fix typos and grammar

### PR DESCRIPTION
**Pull request by greggdourgarian**
fix typos and grammar

btw, i love Parity...can i have a job?

### CODE REVIEW

**Comment by gavofyork** [comment]
sure - send a cv to info@parity.io

File: js/src/i18n/_default/settings.js [hunk 1]
```diff
@@ -57,17 +57,17 @@ export default {
...
     contracts: {
       description: `Watch and interact with specific contracts that have been deployed on the network. This is a more technically-focused environment, specifically for advanced users that understand the inner working of certain contracts.`,
       label: `Contracts`
     },
-    overview_0: `Manage the available application views, using only the parts of the application that is applicable to you.`,
-    overview_1: `Are you an end-user? The defaults are setups for both beginner and advanced users alike.`,
-    overview_2: `Are you a developer? Add some features to manage contracts are interact with application deployments.`,
-    overview_3: `Are you a miner or run a large-scale node? Add the features to give you all the information needed to watch the node operation.`,
+    overview_0: `Manage the available application views, using only the parts of the application applicable to you.`,
+    overview_1: `Are you an end-user? The defaults are setup for both beginner and advanced users alike.`,
```
Comment (rphmeier) [inline js/src/i18n/_default/settings.js]: To nitpick further, should be "set up"

Comment (rphmeier) [inline js/src/i18n/_default/settings.js]: actually, this wasn't wrong before: a "setup" is a real noun, so "the defaults are setups (that ...)" is fine

File: js/src/views/Settings/Views/views.js [hunk 2]
```diff
@@ -45,19 +45,19 @@ class Views extends Component {
...
                 id='settings.views.overview_0'
-                defaultMessage='Manage the available application views, using only the parts of the application that is applicable to you.'
+                defaultMessage='Manage the available application views using only the parts of the application applicable to you.'
               />
             </div>
             <div>
               <FormattedMessage
                 id='settings.views.overview_1'
-                defaultMessage='Are you an end-user? The defaults are setups for both beginner and advanced users alike.'
+                defaultMessage='Are you an end-user? The defaults are setup for both beginner and advanced users alike.'
```
Comment (rphmeier) [inline js/src/views/Settings/Views/views.js]: same comments apply here

**Comment by greggdourgarian** [comment]
?wow u guys burn the midnight oil...tx for merging so quickly.

________________________________
________________________________

**Pull request merged**